src/test/java/AndroidTest.java

```diff
@@ -157,6 +157,11 @@ public void test5(){
     public void test6(){
         //test code here
     }
+    @Test
+
+    public void test7(){
```

Comment:
This is https://github.com/johnow2001/Android/pull/7
Need to
- [ ] Run test
- [ ] Review test
- [ ] Debug